CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Unreleased
 
+## v0.9.3
+### September 7, 2023
```

Comment:
Very nitty nit: Normally HashiCorp changelogs have the date inlined, and some markdown parsers are a bit picky about newlines being required after headings.

```suggestion
## v0.9.3 (September 7, 2023)

```